src/Tribe/Attendees.php

```diff
@@ -140,11 +140,59 @@ public function event_action_links( $event_id ) {
 	 * Print Check In Totals at top of Column.
 	 *
 	 * @since 4.6.2
+	 * @since TBD   Added $post_id parameter.
 	 */
-	public function print_checkedin_totals() {
-		$total_checked_in = Tribe__Tickets__Main::instance()->attendance_totals()->get_total_checked_in();
+	public function print_checkedin_totals( $post_id ) {
+		// Bail if we don't have a post ID.
+		if ( ! $post_id ) {
+			return;
+		}
+
+		$total_checked_in = $this->get_checkedin_total();
+		$check_in_percent = $this->get_checkedin_percentage( $post_id );
+		$total_attendees  = Tribe__Tickets__Tickets::get_event_attendees_count( $post_id );
+
+		/** @var Tribe__Tickets__Admin__Views $admin_views */
+		$admin_views = tribe( 'tickets.admin.views' );
+		$args = [
+			'total_attendees'    => absint( $total_attendees ),
+			'total_checked_in'   => absint( $total_checked_in ),
+			'percent_checked_in' => $check_in_percent,
+		];
+		$admin_views->template( 'attendees/attendees-event/attendance-totals', $args, true );
+	}
+
+	/**
+	 * Get Check In Total.
+	 *
+	 * @since TBD
+	 * 
+	 * @return int
+	 */
+	public function get_checkedin_total(): int {
+		return (int) Tribe__Tickets__Main::instance()->attendance_totals()->get_total_checked_in();
```

Comment:
Yeah. I just moved this out of `print_checkedin_totals()` and moved it into its own method because we needed to use it within `get_checkedin_percentage()`, too.